drake/systems/plants/collision/ResultCollector.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+#define DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+
+namespace DrakeCollision {
+Eigen::Vector3d toVector3d(const Eigen::Vector3d& vec);
+
+class ResultCollector {
+ public:
+  virtual ~ResultCollector() {}
+
+  virtual void addPointPairResult(const PointPair& result);
```

Comment:
All of these method names violate the naming style guide.  I think this refactoring PR should still be merged without fixing the names -- but please take a note to fix the names in a some future PR.